.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*/node_modules/conventional-changelog-core/.*
```

Comment:
I could do that in a following PR, it would mean we would need to make some changes to the README to tell people that they need to install types when they develop!